tests/errors/semantic/blocking/INSERT_LIST_INCOMPATIBLE_TYPES_0.py

```diff
@@ -0,0 +1,7 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+import numpy as np
+a = [1,2,3]
```

Comment:
Please leave (at least) one blank line after the import statements.

Reply:
fixed in cfa5a2a986d745a91d730022bb3f166c06a32311